### ISSUE TITLE
Ctrl-C doesn't close the message-tool.

### ISSUE THREAD

**Issue by nimrodshn** [container-mgmt/messaging-library#72]
Pressing Ctrl-C doesn't close the message-server.

cc: @yaacov @zgalor

**Comment by yaacov** [comment]
I'm assigning this to you since you can reproduce and solve it.

**Comment by nimrodshn** [comment]
@yaacov @zgalor also reproduced this.. if you don't want to handle it that's fine i'll close it.

**Comment by yaacov** [comment]
> @zgalor also reproduced this.. if you don't want to handle it that's fine i'll close it.

@nimrodshn  I want you ( with @zgalor help, if he can also reproduce ) to handle it.

I can not handle it because ctrl-c breaks the server on my machines ( home and work )

```
~/go/src/github.com/container-mgmt/messaging-library$ .gopath/bin/messaging-server serve 
I0704 13:34:05.036245   20502 serve.go:71] Your friendly STOMP broker (YFSB) server 127.0.0.1:61613: [listen to all]
^C
~/go/src/github.com/container-mgmt/messaging-library$
```

**Comment by nimrodshn** [comment]
@yaacov Please check if Ctrl-C works on `messaging-tool`?
I got confused between messaging-tool and messaging-server.

**Comment by yaacov** [comment]
Ahh, we know that ... you can fix it if you want ...

**Comment by nimrodshn** [comment]
Working on it with @zgalor  :+1:

**Comment by yaacov** [comment]
closed by #73
